Packages/vcs/Lib/Canvas.py

```diff
@@ -2679,11 +2679,25 @@ def plot_filledcontinents(self,slab,template_name,g_type,g_name,bg,ratio):
         except Exception,err:
             print err
 
+    def __new_elts(self,original,new):
```

Comment:
when I do the vcs pep8, then I will rebase this on top of it and do the pep8 on this too. You guys can approve it then. That's fine with me.